src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializerOptions.Converters.cs

```diff
@@ -46,8 +47,6 @@ static JsonTypeInfo CreateJsonTypeInfo(Type type, JsonSerializerOptions options)
                 JsonTypeInfo.ValidateType(type, null, null, options);
```

Comment:
Nit: probably fine for the scope of this PR, but eventually I'd like to see this method moved out of `JsonSerializerOptions` responsibility.